zkp-toolkit/scheme/src/bulletproofs/arithmetic_circuit.rs

```diff
@@ -844,7 +844,10 @@ mod tests {
         }
         .matrix_to_map();
 
-        let f = [&statement[..], &witness[..]].concat();
+        let mut r1_statement = vec![G::Fr::one()];
```

Comment:
Mark: we put `one` in the first of the `statement` array. It acts as a default public input.